python/mxnet/_numpy_op_doc.py

```diff
@@ -176,3 +173,38 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         `axis` is not None or `a` is a 1-d array.
     """
     pass
+
+def _np_broadcast_to(array, shape, subok=False):
+    r"""
+    _np_broadcast_to(array, shape)
```

Comment:
For overriding the signature, you just need `broadcast_to(array, shape)`

Reply:
ok